ofborg/src/tasks/githubcommentposter.rs

```diff
@@ -151,6 +151,14 @@ fn result_to_check(result: &LegacyBuildResult, timestamp: DateTime<Utc>) -> Chec
         ));
     }
 
+    if let Some(up) = result.cachix_uploaded {
+        if up {
```

Comment:
More nits, but I'd like the longer and slightly more explicit variable name here... Feel free to disregard.

```suggestion
    if let Some(uploaded) = result.cachix_uploaded {
        if uploaded {
```